cfme/cloud/provider/azure.py

```diff
@@ -8,20 +10,33 @@ class AzureProvider(Provider):
     mgmt_class = AzureSystem
 
     def __init__(self, name=None, credentials=None, zone=None, key=None, region=None,
-                 tenant_id=None, subscription_id=None):
+                 tenant_id=None, subscription_id=None, provisioning=None):
         super(AzureProvider, self).__init__(name=name, credentials=credentials,
                                             zone=zone, key=key)
-        self.region = region
+        self.provisioning = provisioning
         self.tenant_id = tenant_id
         self.subscription_id = subscription_id
 
     def _form_mapping(self, create=None, **kwargs):
-        # Will still need to figure out where to put the tenant id.
-        return {'name_text': kwargs.get('name'),
-                'type_select': create and 'Azure',
-                'region_select': kwargs.get('region'),
-                'azure_tenant_id': kwargs.get('tenant_id'),
-                'azure_subscription_id': kwargs.get('subscription_id')}
+        data_dict = {
```

Comment:
This is the first time I've done one of these.  Need some eyes on it.  It works, but you may have a different format